CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+
+> IMPORTANT: If using `Excalidraw` to generated diagrams (**recommended**), ensure that when exporting your diagram to `.png` format that you enable the `Embed scene` option. This allows for editing of images that have been generated using Excalidraw, dramatically reducing the amount of effort to update or change the rendered `.png` version of your diagram.
+
+### Embedded Code Guidelines
+
+Code examples should be provided in a complete and runnable state. If showing Python code, for instance, ensure that you are including `pip` install of requirements prior to the first code block, as well as full import statements so that the code within the blog **can be run without any external contextual references**.
+
+If a data source is required to execute the examples, provide the means within the examples of downloading an open-source dataset that is easily accessible (as well as code that will fetch and locally store the data).
+
+Do not inadvertantly commit code that contains sensitive information (e.g., OpenAI keys).
+
+### Spelling and Grammar
+
+Blogs on MLflow use American English spelling and grammar. Please be patient if we ask for grammatical spelling corrections to your post.
+
+### Links
+
+Include links to relevant resources, but avoid promotional content (either organizational or personal) or links to competing tools (we don't want to start a flame war).
+
+When linking to MLflow documentation pages:
+
+* Ensure that you are generating a direct link to the `latest` version of the docs (no relative links or documentation version links). For example: `[transformers](https://www.mlflow.org/docs/latest/llms/transformers/index.html)`
+* Do not repeat links. If you're mentioning a link to, say, [transformers](https://www.mlflow.org/docs/latest/llms/transformers/index.html), only link to the first mention of the term.  
+
+### Accessibility
+
+Aim for accessible writing and formatting, particularly with font sizes in images and overall density within diagrams.
+
+For posted images, include alt text so that screen readers can effectively describe the diagram, image, or video-converted `.gif`.
+
+### First time Contributors
+
+If it's your first time contributing to the MLflow Blog, ensure that you enter a unique entry for yourself (and co-authors, if applicable) within the [authors](website/blog/authors.yml) YAML file. We parse from this collection to populate the links to the authors (including a headshot) so that readers can communicate with you about your contribution. Please ensure that the entries are correct and render well when developing your generated blog locally.
```

Comment:
Do we want to add somewhere, their image is sourced from LinkedIN profile

Reply:
Good call! Added